src/components/MTableToolbar/index.js

```diff
@@ -248,6 +248,10 @@ export function MTableToolbar(props) {
               props.actions &&
               props.actions.filter((a) => a.position === diplayedActions)
             }
+            columns={props.columns}
+            onColumnsChanged={(col, hidden) =>
+              props.onColumnsChanged(col, hidden)
```

Comment:
Canyou add a check if props.onColumnsChanged is not undefined?

Reply:
Done! Let me know if I can do anything more :)

Reply:
@Domino987 do you have any update on this one?